scripts/file-entities.php.in

```diff
@@ -216,16 +216,21 @@ function file_entities($work_dir, $trans_dir, $orig_dir, &$entities, $prefix=fal
 
         } // end of "if xml file"
     } // end of readdir loop
-    
+
     // Close directory
     closedir($dh);
 
     // If we created a function entities list, write it out
     if (isset($function_entities)) {
-        
+
         // Sort by name
         sort($function_entities);
-        
+
+        $function_entities = array_map(
+            fn ($entity): string => $entity . ";",
```

Comment:
Requiring PHP 7.4 is probably not an issue, but the requirement seems not to be mentioned in README.md. Is it maybe somewhere else?

Reply:
The only requirement I've seen is in `configure.php` which has an explicit check for PHP 5.0. Unfortunately I have already messed that up by adding a return type in another PR. Do you need me to open a PR to remove that return type?

As far as this PR goes, I'll go with what @jimwins suggested above and add the trailing semicolon to the `fputs()` call.

Reply:
Regarding the PHP 7.4 requirement: this is fine for me. Maybe @Girgias has something to say about this.

Reply:
The doc machine now runs at least PHP 8.0 as I got Derick to upgrade the PHP version. So I am happy to require 7.4